.gitignore

```diff
@@ -34,4 +34,6 @@ yarn-error.log*
 
 *.log
 
-node_modules/
+schema.graphql
```

Comment:
Prefer to commit the schema, so you don't need to generate it every time you need to pull a commit and so on.

Also, it's not something like generated files from Relay that is not useful for a repository, but schema can help in most cases when you need a summary of the backend.